test/integration/static_assets/test_application.rb

```diff
@@ -81,7 +97,7 @@
 JS
       end
 
-      get '/assets/dashboard.js'
+      get 'assets/dashboard.js'
```

Comment:
I did it for checking problem when you write routes as like this `src="assets/application.js"`